qa/restart-upgrade/build.gradle

```diff
@@ -76,40 +76,40 @@ task testAgainstOldCluster(type: StandaloneRestIntegTestTask) {
     // because these features were released in 2.11 version.
     if (versionsBelow2_11.any { ext.neural_search_bwc_version.startsWith(it) }){
         filter {
-            excludeTestsMatching "org.opensearch.neuralsearch.bwc.MultiModalSearchIT.*"
-            excludeTestsMatching "org.opensearch.neuralsearch.bwc.HybridSearchIT.*"
-            excludeTestsMatching "org.opensearch.neuralsearch.bwc.NeuralSparseSearchIT.*"
-            excludeTestsMatching "org.opensearch.neuralsearch.bwc.NeuralQueryEnricherProcessorIT.*"
+            excludeTestsMatching "org.opensearch.neuralsearch.bwc.restart.MultiModalSearchIT.*"
```

Comment:
Can you also do the similar change rolling upgrade build.gradle

Reply:
There is no such thing for rolling upgrade build.gradle in main branch. I need to make such change when I backport it to 2.x